src/snap-confine.apparmor.in

```diff
@@ -56,23 +58,24 @@
 
     # don't allow changing profile to unconfined or profiles that start with
     # '/'
```

Comment:
Can you adjust this comment to be:
```
# Don't allow changing profile to unconfined or profiles that start with
￼# '/'. Use 'unsafe' to support snap-exec on armhf and its reliance on
# the environment for determining the capabilities of the architecture.
# 'unsafe' is ok here because the kernel will have already cleared the
# environment as part of launching snap-confine (since it has
# CAP_SYS_ADMIN).
```